restifizer/RestifizerResponse.h

```diff
@@ -0,0 +1,22 @@
+//
+//  RestifizerResponce.h
+//  Restifizer
+//
+//  Created by Nikita Pankiv on 4/29/16.
+//  Copyright © 2016 VedideV. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "RestifizerError.h"
+
+@class RestifizerRequest;
+
+@interface RestifizerResponse : NSObject
+
+@property (nonatomic, strong) NSData *contentData;
```

Comment:
you can perform JSON-decoding there https://github.com/vedidev/restifizer-ios/pull/1/files#diff-ce50aebca702d29e770efc9d80b652adR242